nuget/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.Package.csproj

```diff
@@ -14,15 +14,13 @@
   
   <ItemGroup>
     <AnalyzerNupkgDependency Include="Microsoft.CodeQuality.Analyzers" />
-    <AnalyzerNupkgDependency Include="Microsoft.CodeQuality.Analyzers.Exp" />
```

Comment:
This is a port from [2.6.1](https://github.com/dotnet/roslyn-analyzers/tree/2.6.1) branch that was used to generate the [latest release](https://github.com/dotnet/roslyn-analyzers/releases/tag/v2.6.1)